types/hot-formula-parser/index.d.ts

```diff
@@ -0,0 +1,555 @@
+    'LARGE',
+    'LCM',
+    'LEFT',
+    'LEN',
+    'LINEST',
+    'LN',
+    'LOG',
+    'LOG10',
+    'LOGEST',
+    'LOGNORM.DIST',
+    'LOGNORM.INV',
+    'LOGNORMDIST',
+    'LOGNORMINV',
+    'LOWER',
+    'LT',
+    'LTE',
+    'MATCH',
+    'MAX',
+    'MAXA',
+    'MEDIAN',
+    'MID',
+    'MIN',
+    'MINA',
+    'MINUS',
+    'MINUTE',
+    'MIRR',
+    'MOD',
+    'MODE.MULT',
+    'MODE.SNGL',
+    'MODEMULT',
+    'MODESNGL',
+    'MONTH',
+    'MROUND',
+    'MULTINOMIAL',
+    'MULTIPLY',
+    'NE',
+    'NEGBINOM.DIST',
+    'NEGBINOMDIST',
+    'NETWORKDAYS',
+    'NOMINAL',
+    'NORM.DIST',
+    'NORM.INV',
+    'NORM.S.DIST',
+    'NORM.S.INV',
+    'NORMDIST',
+    'NORMINV',
+    'NORMSDIST',
+    'NORMSINV',
+    'NOT',
+    'NOW',
+    'NPER',
+    'NPV',
+    'NUMBERS',
+    'OCT2BIN',
+    'OCT2DEC',
+    'OCT2HEX',
+    'ODD',
+    'OR',
+    'PDURATION',
+    'PEARSON',
+    'PERCENTILEEXC',
+    'PERCENTILEINC',
+    'PERCENTRANKEXC',
+    'PERCENTRANKINC',
+    'PERMUT',
+    'PERMUTATIONA',
+    'PHI',
+    'PI',
+    'PMT',
+    'POISSON.DIST',
+    'POISSONDIST',
+    'POW',
+    'POWER',
+    'PPMT',
+    'PROB',
+    'PRODUCT',
+    'PROPER',
+    'PV',
+    'QUARTILE.EXC',
+    'QUARTILE.INC',
+    'QUARTILEEXC',
+    'QUARTILEINC',
+    'QUOTIENT',
+    'RADIANS',
+    'RAND',
+    'RANDBETWEEN',
+    'RANK.AVG',
+    'RANK.EQ',
+    'RANKAVG',
+    'RANKEQ',
+    'RATE',
+    'REFERENCE',
+    'REGEXEXTRACT',
+    'REGEXMATCH',
+    'REGEXREPLACE',
+    'REPLACE',
+    'REPT',
+    'RIGHT',
+    'ROMAN',
+    'ROUND',
+    'ROUNDDOWN',
+    'ROUNDUP',
+    'ROW',
+    'ROWS',
+    'RRI',
+    'RSQ',
+    'SEARCH',
+    'SEC',
+    'SECH',
+    'SECOND',
+    'SERIESSUM',
+    'SIGN',
+    'SIN',
+    'SINH',
+    'SKEW',
+    'SKEW.P',
+    'SKEWP',
+    'SLN',
+    'SLOPE',
+    'SMALL',
+    'SPLIT',
+    'SPLIT',
+    'SQRT',
+    'SQRTPI',
+    'STANDARDIZE',
+    'STDEV.P',
+    'STDEV.S',
+    'STDEVA',
+    'STDEVP',
+    'STDEVPA',
+    'STDEVS',
+    'STEYX',
+    'SUBSTITUTE',
+    'SUBTOTAL',
+    'SUM',
+    'SUMIF',
+    'SUMIFS',
+    'SUMPRODUCT',
+    'SUMSQ',
+    'SUMX2MY2',
+    'SUMX2PY2',
+    'SUMXMY2',
+    'SWITCH',
+    'SYD',
+    'T',
+    'T.DIST',
+    'T.DIST.2T',
+    'T.DIST.RT',
+    'T.INV',
+    'T.INV.2T',
+    'TAN',
+    'TANH',
+    'TBILLEQ',
+    'TBILLPRICE',
+    'TBILLYIELD',
+    'TDIST',
+    'TDIST2T',
+    'TDISTRT',
+    'TIME',
+    'TIMEVALUE',
+    'TINV',
+    'TINV2T',
+    'TODAY',
+    'TRANSPOSE',
+    'TREND',
+    'TRIM',
+    'TRIMMEAN',
+    'TRUE',
+    'TRUNC',
+    'UNICHAR',
+    'UNICODE',
+    'UNIQUE',
+    'UPPER',
+    'VAR.P',
+    'VAR.S',
+    'VARA',
+    'VARP',
+    'VARPA',
+    'VARS',
+    'WEEKDAY',
+    'WEEKNUM',
+    'WEIBULL.DIST',
+    'WEIBULLDIST',
+    'WORKDAY',
+    'XIRR',
+    'XNPV',
+    'XOR',
+    'YEAR',
+    'YEARFRAC',
+];
+
+export const BUILT_IN_VARIABLES: readonly ['TRUE', 'FALSE', 'NULL'];
```

Comment:
I don't see this in the [source](https://github.com/handsontable/formula-parser/blob/develop/src/index.js).  Moreover, the docs on `setVariable` don't seem accurate - you can [easily redefine these](https://github.com/handsontable/formula-parser/blob/4ef971bfa550af631d1c79636455c4cb7730123d/src/parser.js#L81).

Reply:
Well noticed, thank you!
Both `setVariable` and `setFunction` return the `Parser` instance.
Built-in variables aren't exported by the package, so I removed them as well. However, I've added a mention of them in the `setVariable` method, as knowing about them might be helpful.